Makefile

```diff
@@ -280,6 +287,11 @@ generate-go: $(CONTROLLER_GEN) $(MOCKGEN) $(CONVERSION_GEN) $(KUBEBUILDER) $(KUS
 		--output-file-base=zz_generated.conversion  $(CONVERSION_GEN_OUTPUT_BASE) \
 		--go-header-file=./hack/boilerplate/boilerplate.generatego.txt
 
+	$(CONVERSION_GEN) \
+		--input-dirs=./api/v1alpha5 \
+		--output-file-base=zz_generated.conversion  $(CONVERSION_GEN_OUTPUT_BASE) \
+		--go-header-file=./hack/boilerplate/boilerplate.generatego.txt
+
```

Comment:
Why do we need this conversion when we are already in the release-0.5 branch? 
In main we [do](https://github.com/metal3-io/cluster-api-provider-metal3/blob/main/Makefile#L318-L321) conversion gen to v1alpha5 (we are in v1beta1), in release-0.4 branch we [do](https://github.com/metal3-io/cluster-api-provider-metal3/blob/release-0.4/Makefile#L267-L270) conversion gen to v1alpha3 (we are in v1alpha4). But here we are in release-0.5 branch already and doing conversion to v1alpha5?

Reply:
Yes you are right, It should be conversion gen to v1alpha4 and conversion to v1alpha5 should not be there. I am removing it.